roms.sha1

```diff
@@ -3,3 +3,4 @@ f2f52230b536214ef7c9924f483392993e226cfb *pokecrystal11.gbc
 a0fc810f1d4e124434f7be2c989ab5b5892ddf36 *pokecrystal_au.gbc
 c60d57a24bbe8ecf7cba54ab3f90669f97bd330d *pokecrystal_debug.gbc
 391ae86b1d5a26db712ffe6c28bbf2a1f804c3c4 *pokecrystal11_debug.gbc
+a25517f60ca0e887d39ec698aa56a0040532a4b3  pokecrystalvc.patch
```

Comment:
```suggestion
a25517f60ca0e887d39ec698aa56a0040532a4b3 *pokecrystalvc.patch
```
Checksums should be in binary mode, in case text mode ignores LF/CRLF differences (GNU does not).

Reply:
I thought the `*` was just a result of the file being marked as executable on the filesystem.